src/main/java/io/storj/StorjException.java

```diff
@@ -3,7 +3,7 @@
 /**
  * Signals that an exception occurred when calling a method of the Storj library.
  */
-public class StorjException extends Exception {
+public class StorjException extends RuntimeException {
```

Comment:
Checked exception are not very effective, especially with lambdas. There are strong opinions that checked exceptions should be avoided.

And I couldn't see any advantage of checked exceptions if we through the same exception everywhere. (We already wrap the exception with `RuntimExecption` in the iterators):

http://literatejava.com/exceptions/checked-exceptions-javas-biggest-mistake/

https://testing.googleblog.com/2009/09/checked-exceptions-i-love-you-but-you.html

I modified it to make the usage / example more simple.

But TBH I argued so many times to avoid checked exceptions that I am tired, if anybody would like to have it I am fine with keeping it...